README.md

```diff
@@ -2,21 +2,19 @@
 
 Public documentation and sample code for integration with Connected Cars
 
-## Integrations options
+## Integrations
 
-### Push
+List of common integrations for markets.
 
-The push api provides a close to real time data steam for all the collected parameters.
-
-[Read more here](./push.md)
+[Read more here](./integrations/intro)
```

Comment:
```suggestion
[Read more here](./integrations/intro.md)
```

Reply:
It's not needed: https://docs.connectedcars.io/#/./integrations/intro

Reply:
Ah alright, I just assumed since it was present in all the other links.